DiffuseTests/DiffuseComparatorTests.swift

```diff
@@ -5,13 +5,42 @@
 import XCTest
 @testable import Diffuse
 
-class DiffuseTests: XCTestCase {
+class DiffuseComparatorTests: XCTestCase {
     struct Object: Equatable {
         let objId: Int
         var name: String
     }
 
-    // MARK: - Primitives
+    func testEmptyOldEmptyNew() {
+        let old = [Int]()
+        let new = [Int]()
+        let changes = Diffuse.diff(old: old, new: new, comparator: { $0 == $1 })
+
+        // Both collections are empty.
+        XCTAssertEqual(0, changes.count)
+    }
+
+    // MARK: - Comparing primitives
+
+    func testEmptyOld_withPrimitives() {
```

Comment:
I would try to avoid using underscores for naming in iOS

Reply:
I agree. The names of these testing functions aren't long enough (yet) to justify the underscore. Changed